src/libraries/Microsoft.Extensions.Hosting/ref/Microsoft.Extensions.Hosting.cs

```diff
@@ -30,6 +30,12 @@ public static partial class Host
         public static Microsoft.Extensions.Hosting.IHostBuilder CreateDefaultBuilder() { throw null; }
         public static Microsoft.Extensions.Hosting.IHostBuilder CreateDefaultBuilder(string[]? args) { throw null; }
     }
+    public sealed partial class HostAbortedException : System.Exception
```

Comment:
Yes I did. Why does the generator not generate the Serializable Attribute?

Reply:
Maybe I also deleted a bit too much of the newly generated during the commit. I'll regenerate it this afternoon (CEST).

Reply:
Looking around at other code, we don't have it on any `ref` file, so the tooling must automatically exclude it. Since we don't have it anywhere else, we don't need it here.

Reply:
@eerhardt 

The ref assembly generator generated this just so.
```
E:\external\dotnet\runtime\src\libraries\Microsoft.Extensions.Hosting\src>dotnet msbuild /t:GenerateReferenceAssemblySource /p:Configuration=Release
Microsoft (R)-Build-Engine, Version 17.3.0-preview-22226-04+f15ed2652 für .NET
Copyright (C) Microsoft Corporation. Alle Rechte vorbehalten.

  Microsoft.Interop.SourceGeneration -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Interop.SourceGeneration\Rel
  ease\netstandard2.0\Microsoft.Interop.SourceGeneration.dll
  Microsoft.Extensions.Logging.Generators.Roslyn3.11 -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.L
  ogging.Generators.Roslyn3.11\Release\netstandard2.0\Microsoft.Extensions.Logging.Generators.dll
  Microsoft.Extensions.Logging.Generators.Roslyn4.0 -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Lo
  gging.Generators.Roslyn4.0\Release\netstandard2.0\Microsoft.Extensions.Logging.Generators.dll
  System.Runtime -> E:\external\dotnet\runtime\artifacts\bin\System.Runtime\ref\Release\net7.0\System.Runtime.dll
  System.Text.Encodings.Web -> E:\external\dotnet\runtime\artifacts\bin\System.Text.Encodings.Web\ref\Release\net7.0\Sy
  stem.Text.Encodings.Web.dll
  System.Diagnostics.EventLog -> E:\external\dotnet\runtime\artifacts\bin\System.Diagnostics.EventLog\ref\Release\net7.
  0\System.Diagnostics.EventLog.dll
  LibraryImportGenerator -> E:\external\dotnet\runtime\artifacts\bin\LibraryImportGenerator\Release\netstandard2.0\Micr
  osoft.Interop.LibraryImportGenerator.dll
  System.Text.Encodings.Web -> E:\external\dotnet\runtime\artifacts\bin\System.Text.Encodings.Web\Release\net7.0\System
  .Text.Encodings.Web.dll
  Microsoft.Extensions.Logging.Abstractions -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Logging.Ab
  stractions\ref\Release\net7.0\Microsoft.Extensions.Logging.Abstractions.dll
  Microsoft.Extensions.DependencyInjection.Abstractions -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extension
  s.DependencyInjection.Abstractions\ref\Release\net7.0\Microsoft.Extensions.DependencyInjection.Abstractions.dll
  Microsoft.Extensions.FileSystemGlobbing -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.FileSystemGl
  obbing\ref\Release\net7.0\Microsoft.Extensions.FileSystemGlobbing.dll
  Microsoft.Extensions.Primitives -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Primitives\ref\Relea
  se\net7.0\Microsoft.Extensions.Primitives.dll
  Microsoft.Extensions.Primitives -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Primitives\Release\n
  et7.0\Microsoft.Extensions.Primitives.dll
  System.Diagnostics.EventLog -> E:\external\dotnet\runtime\artifacts\bin\System.Diagnostics.EventLog\Release\net7.0\Sy
  stem.Diagnostics.EventLog.dll
  Microsoft.Extensions.FileProviders.Abstractions -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.File
  Providers.Abstractions\ref\Release\net7.0\Microsoft.Extensions.FileProviders.Abstractions.dll
  Microsoft.Extensions.Configuration.Abstractions -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Conf
  iguration.Abstractions\ref\Release\net7.0\Microsoft.Extensions.Configuration.Abstractions.dll
  Microsoft.Extensions.FileSystemGlobbing -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.FileSystemGl
  obbing\Release\net7.0\Microsoft.Extensions.FileSystemGlobbing.dll
  Microsoft.Extensions.Logging.Abstractions -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Logging.Ab
  stractions\Release\net7.0\Microsoft.Extensions.Logging.Abstractions.dll
  System.Text.Json.SourceGeneration.Roslyn4.0 -> E:\external\dotnet\runtime\artifacts\bin\System.Text.Json.SourceGenera
  tion.Roslyn4.0\Release\netstandard2.0\System.Text.Json.SourceGeneration.dll
  System.Text.Json.SourceGeneration.Roslyn3.11 -> E:\external\dotnet\runtime\artifacts\bin\System.Text.Json.SourceGener
  ation.Roslyn3.11\Release\netstandard2.0\System.Text.Json.SourceGeneration.dll
  Microsoft.Extensions.DependencyInjection.Abstractions -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extension
  s.DependencyInjection.Abstractions\Release\net7.0\Microsoft.Extensions.DependencyInjection.Abstractions.dll
  Microsoft.Extensions.Configuration.Abstractions -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Conf
  iguration.Abstractions\Release\net7.0\Microsoft.Extensions.Configuration.Abstractions.dll
  Microsoft.Extensions.FileProviders.Abstractions -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.File
  Providers.Abstractions\Release\net7.0\Microsoft.Extensions.FileProviders.Abstractions.dll
  Microsoft.Extensions.Options -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Options\ref\Release\net
  7.0\Microsoft.Extensions.Options.dll
  Microsoft.Extensions.Configuration -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Configuration\ref
  \Release\net7.0\Microsoft.Extensions.Configuration.dll
  Microsoft.Extensions.DependencyInjection -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.DependencyI
  njection\ref\Release\net7.0\Microsoft.Extensions.DependencyInjection.dll
  Microsoft.Extensions.FileProviders.Physical -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.FileProv
  iders.Physical\ref\Release\net7.0\Microsoft.Extensions.FileProviders.Physical.dll
  Microsoft.Extensions.Hosting.Abstractions -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Hosting.Ab
  stractions\ref\Release\net7.0\Microsoft.Extensions.Hosting.Abstractions.dll
  System.Memory -> E:\external\dotnet\runtime\artifacts\bin\System.Memory\ref\Release\net7.0\System.Memory.dll
  Microsoft.Extensions.Configuration -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Configuration\Rel
  ease\net7.0\Microsoft.Extensions.Configuration.dll
  Microsoft.Extensions.DependencyInjection -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.DependencyI
  njection\Release\net7.0\Microsoft.Extensions.DependencyInjection.dll
  Microsoft.Extensions.Configuration.Binder -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Configurat
  ion.Binder\ref\Release\net7.0\Microsoft.Extensions.Configuration.Binder.dll
  Microsoft.Extensions.Options -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Options\Release\net7.0\
  Microsoft.Extensions.Options.dll
  Microsoft.Extensions.Configuration.EnvironmentVariables -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensi
  ons.Configuration.EnvironmentVariables\ref\Release\net7.0\Microsoft.Extensions.Configuration.EnvironmentVariables.dll
  Microsoft.Extensions.Configuration.CommandLine -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Confi
  guration.CommandLine\ref\Release\net7.0\Microsoft.Extensions.Configuration.CommandLine.dll
  Microsoft.Extensions.Configuration.Binder -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Configurat
  ion.Binder\Release\net7.0\Microsoft.Extensions.Configuration.Binder.dll
  System.Collections.Concurrent -> E:\external\dotnet\runtime\artifacts\bin\System.Collections.Concurrent\ref\Release\n
  et7.0\System.Collections.Concurrent.dll
  Microsoft.Extensions.Logging -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Logging\ref\Release\net
  7.0\Microsoft.Extensions.Logging.dll
  System.Collections -> E:\external\dotnet\runtime\artifacts\bin\System.Collections\ref\Release\net7.0\System.Collectio
  ns.dll
  System.Text.Json -> E:\external\dotnet\runtime\artifacts\bin\System.Text.Json\ref\Release\net7.0\System.Text.Json.dll
  Microsoft.Extensions.Options.ConfigurationExtensions -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions
  .Options.ConfigurationExtensions\ref\Release\net7.0\Microsoft.Extensions.Options.ConfigurationExtensions.dll
  Microsoft.Extensions.FileProviders.Physical -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.FileProv
  iders.Physical\Release\net7.0\Microsoft.Extensions.FileProviders.Physical.dll
  Microsoft.Extensions.Logging -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Logging\Release\net7.0\
  Microsoft.Extensions.Logging.dll
  Microsoft.Extensions.Options.ConfigurationExtensions -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions
  .Options.ConfigurationExtensions\Release\net7.0\Microsoft.Extensions.Options.ConfigurationExtensions.dll
  Microsoft.Extensions.Logging.Debug -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Logging.Debug\ref
  \Release\net7.0\Microsoft.Extensions.Logging.Debug.dll
  Microsoft.Extensions.Configuration.FileExtensions -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Co
  nfiguration.FileExtensions\ref\Release\net7.0\Microsoft.Extensions.Configuration.FileExtensions.dll
  Microsoft.Extensions.Configuration.FileExtensions -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Co
  nfiguration.FileExtensions\Release\net7.0\Microsoft.Extensions.Configuration.FileExtensions.dll
  Microsoft.Extensions.Logging.EventLog -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Logging.EventL
  og\ref\Release\net7.0\Microsoft.Extensions.Logging.EventLog.dll
  System.Text.Json -> E:\external\dotnet\runtime\artifacts\bin\System.Text.Json\Release\net7.0\System.Text.Json.dll
  Microsoft.Extensions.Logging.Configuration -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Logging.C
  onfiguration\ref\Release\net7.0\Microsoft.Extensions.Logging.Configuration.dll
  Microsoft.Extensions.Logging.EventSource -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Logging.Eve
  ntSource\ref\Release\net7.0\Microsoft.Extensions.Logging.EventSource.dll
  Microsoft.Extensions.Configuration.Json -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Configuratio
  n.Json\ref\Release\net7.0\Microsoft.Extensions.Configuration.Json.dll
  Microsoft.Extensions.Logging.Configuration -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Logging.C
  onfiguration\Release\net7.0\Microsoft.Extensions.Logging.Configuration.dll
  Microsoft.Extensions.Logging.Console -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Logging.Console
  \ref\Release\net7.0\Microsoft.Extensions.Logging.Console.dll
  Microsoft.Extensions.Configuration.Json -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Configuratio
  n.Json\Release\net7.0\Microsoft.Extensions.Configuration.Json.dll
  Microsoft.Extensions.Configuration.UserSecrets -> E:\external\dotnet\runtime\artifacts\bin\Microsoft.Extensions.Confi
  guration.UserSecrets\ref\Release\net7.0\Microsoft.Extensions.Configuration.UserSecrets.dll
  Generated reference assembly source code: E:\external\dotnet\runtime\src\libraries\Microsoft.Extensions.Hosting\ref\M
  icrosoft.Extensions.Hosting.cs
```

Only some attribute-diffs
![grafik](https://user-images.githubusercontent.com/10373452/169360914-806f0683-bdf9-43d3-8cfe-7de93c186d46.png)

Reply:
Oh you marked this as resolved. Right?

Reply:
> Oh you marked this as resolved. Right?

Correct - no other "ref" files have Serializable attributes on them, so this one doesn't need it either. I resolved the conversation to indicate this is no longer a concern.